cli/clearCache.js

```diff
@@ -21,7 +21,7 @@ function clearJestCache() {
   }
 }
 
-function clearJestPreviewCache() {
+function clearJestPreviewCache () {
```

Comment:
Can you tell me which rule of eslint is this?

Reply:
Added linting support using eslint and fixing errors.